xmake/modules/detect/tools/find_emxx.lua

```diff
@@ -40,6 +41,17 @@ function main(opt)
     -- init options
     opt = opt or {}
 
+    -- init the search directories
+    local paths = {}
+    local emscriptendir = find_emscripten()
+    if emscriptendir then
+        table.insert(paths, emscriptendir.sdkdir)
+    end
+
+    -- find program
+    opt.paths = paths
+    opt.envs  = {PATH = os.getenv("PATH")}
```

Comment:
why? find_program will find it from $PATH by default.

Reply:
didn't know that, I took it from find_vcpkg